docs/user/admin/monitoring.rst

```diff
@@ -0,0 +1,59 @@
+.. highlight:: sh
+
+=================
+Plugin Monitoring
+=================
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Introduction
+============
+
+By a stats endpoint, you are able to collect metrics for the plugin within the interval. Note that only node level statistics collecting is implemented for now. In other words, you only get the metrics for the node you're accessing. Cluster level statistics is yet to implement.
```

Comment:
Not sure how strict we want to be on correct grammar and semantics- but in the interest of insisting on the highest standards, the last sentence should be worded something like "Cluster level statistics have yet to be implemented."

Reply:
Sure, will do. Thanks!